rules/aws_s3_no_global_endpoint.go

```diff
@@ -0,0 +1,68 @@
+package rules
+
+import (
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-aws/project"
+)
+
+// AwsS3NoGlobalEndpointRule checks whether deprecated s3 global endpoint is used instead
+// of the regional endoint
+type AwsS3NoGlobalEndpointRule struct {
+	tflint.DefaultRule
+
+	// TODO: do we need this, if so why
+	resourceType  string
+	attributeName string
+}
+
+// NewAwsS3NoGlobalEndpointRule returns new rule with default attributes
+func NewAwsS3NoGlobalEndpointRule() *AwsS3NoGlobalEndpointRule {
+	return &AwsS3NoGlobalEndpointRule{
+		resourceType:  "aws_s3_bucket",
+		attributeName: "",
+	}
+}
+
+// Name returns the rule name
+func (r *AwsS3NoGlobalEndpointRule) Name() string {
+	return "aws_acm_certificate_lifecycle"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *AwsS3NoGlobalEndpointRule) Enabled() bool {
+	return true
```

Comment:
Given that the attribute isn't deprecated in the provider itself, this definitely should not be enabled by default. Terraform will automatically print warnings when deprecated attributes are referenced so if there's truly no need for this today except for exceptional backwards compatibility cases it should probably be deprecated upstream too. Not a requirement for merging this rule but worth kicking off given that it's a 1 line change.